packages/eslint-plugin/tests/rules/method-signature-style.test.ts

```diff
@@ -348,5 +348,28 @@ interface Foo {
         },
       ],
     },
+    {
+      code: noFormat`
+      declare global {
+        namespace jest {
+          interface Matchers<R, T> {
+            // Add overloads specific to the DOM
+            toHaveProp<K extends keyof DomPropsOf<T>>(name: K, value?: DomPropsOf<T>[K]): R;
+            toHaveProps(props: Partial<DomPropsOf<T>>): R;
+          }
+        }
+      }
+      `,
```

Comment:
indent the example one line, and then you can use the fixer to auto-format the code block
```suggestion
      code: `
        declare global {
          namespace jest {
            interface Matchers<R, T> {
              // Add overloads specific to the DOM
              toHaveProp<K extends keyof DomPropsOf<T>>(name: K, value?: DomPropsOf<T>[K]): R;
              toHaveProps(props: Partial<DomPropsOf<T>>): R;
            }
          }
        }
      `,
```